content/bots/twitterbots/_sys_bot_es.md

```diff
@@ -2,7 +2,7 @@
 Title: @_sys_bot_
```

Comment:
@hectorpalmatellez That's fine, I can add it. Thank you! 

The current URL bug I just mentioned seems to be a pretty big deal, still investigating.